doc/decision-services-arm-template-deployment.md

```diff
@@ -54,8 +54,46 @@ Replace __RESOURCE_GROUP_NAME__ with the name of the Azure Resource Group you wa
 az deployment group create -g RESOURCE_GROUP_NAME --template-file .\InRule.Runtime.DecisionService.json --parameters .\InRule.Runtime.DecisionService.parameters.json
 ```
 
+## Upload valid license file
+
+In order for Decision Services to properly function, you must upload the InRuleLicense.xml file provided to you by InRule to the web app. The simplest way to upload the license file is via the App Service Editor available on the Azure portal. 
+
+First, navigate to the [App Services](https://portal.azure.com/#browse/Microsoft.Web%2Fsites) listing page on the Azure portal and find your newly deployed web app in the list. Click the web app's name to be taken to its' overview page. On the left-hand nav-bar, scroll down until you find the App Service Editor option, under the Development Tools header:
+
+![screen shot of the App Service Editor location on the web app overview page](images/AppServiceEditorScreenshot.png)
+
+On the resulting page, click "Open Editor".
+
+To upload the license file, you will need to drag and drop the InRuleLicense.xml file from your local machine into the top level wwwroot folder. You should be able to see a blue highlight when you are dragging the file into the correct location.
+
+![screen shot of dragging and dropping the license file](images/DragAndDropLicenseScreenshot.png)
+
 ## Verify by getting status details
-Follow the steps to verify rules in the [Decision Services Web App Deployment](decision-services.md#verify-by-getting-status-details)
+As a final verification that Decision Services is properly functioning, a REST call can be made to the status details endpoint.
+
+Get status details from your Decision Services instance:
+```powershell
+#Example: Invoke-RestMethod -Method 'Get' -ContentType 'application/json' -Headers @{"Accept"="application/json"; "inrule-apikey"="SampleApiKey"} -Uri https://contoso-decision-prod-wa.azurewebsites.net/api/status/details
```

Comment:
Not sure we actually need the example

Reply:
Hmm. It was present in the other docs, so I left it here, but I agree that it doesn't seem necessary. I'm happy to remove it if you think we should.